packages/cli/src/bin/pipcook.ts

```diff
@@ -22,6 +23,21 @@ export interface RunOptions {
   npmClient: string;
   // NPM registry
   registry?: string;
+  // Development mode
+  dev: boolean;
+}
+
+export interface PredictOptions {
+  // input for predict
+  str?: string;
+  uri?: string;
+  // Fetch the framework and script without cache
+  nocache?: boolean;
+  // Debug model
+  debug?: boolean;
+  mirror?: string;
+  // Development mode
+  dev?: boolean;
```

Comment:
Shall we avoid this kind of interface, which members are both optional :)

Reply:
One of `uri` and `str` should be passed, they should be optional and we could assert it.
The others have default value, fixed.